libsolidity/ast/AST.cpp

```diff
@@ -181,6 +181,13 @@ void ContractDefinition::setUserDocumentation(Json::Value const& _userDocumentat
 }
 
 
+std::string ContractDefinition::fullyQualifiedName() const
+{
+	std::string sourceString = *(location().sourceName);
```

Comment:
`Declaration`, which should be a base class for this one, already has a function `sourceUnitName()` which does a similar thing, but based on scopes and not on source locations.
Can you add a function `fullyQualifiedName` to the `Declaration` class and return something like `sourceUnitName() + ":" + name()`? I think the `:` should always be present, even if the source unit name is empty.